src/games/strategy/triplea/oddsCalculator/ta/OddsCalculatorPanel.java

```diff
@@ -382,11 +381,7 @@ private boolean isAmphibiousBattle() {
   private Collection<TerritoryEffect> getTerritoryEffects() {
     final Collection<TerritoryEffect> territoryEffects = new ArrayList<>();
     if (m_territoryEffectsJList != null) {
-      final List<Object> selectedObjects = Arrays.asList(m_territoryEffectsJList.getSelectedValuesList());
-      final List<String> selected = new ArrayList<>();
-      for (final Object obj : selectedObjects) {
-        selected.add((String) obj);
-      }
+      final List<String> selected = m_territoryEffectsJList.getSelectedValuesList();
```

Comment:
Are you sure we can make this change without causing issues? I think `selected` needs to be a copy of the list.